Comparison operator does not work between OpCode objects

### Description

Getting this error when using angr library functions which use PyPCode.
```
if op.opcode > pypcode.OpCode.RETURN:
E               TypeError: '>' not supported between instances of 'OpCode' and 'OpCode'

../../../.venv/lib/python3.12/site-packages/angr/engines/pcode/lifter.py:898: TypeError
``` 
This works on PyPCode 2.1.0, breaks on 3.0.0.

### Steps to reproduce the bug

_No response_

### Environment

Using python 3.12, newest version of angr, this also fails on older versions of angr as well.

### Additional context

_No response_

--- a/angr/engines/pcode/lifter.py
+++ b/angr/engines/pcode/lifter.py
@@ -895,10 +895,6 @@ class PcodeBasicBlockLifter:
             last_decode_addr = irsb.addr
             last_imark_idx = 0
             for op_idx, op in enumerate(irsb._ops):
-                # OpCode space begins with control ops ending with RETURN. Quickly filter non-control instructions.
-                if op.opcode > pypcode.OpCode.RETURN:
-                    continue
-
                 if op.opcode == pypcode.OpCode.IMARK:
                     irsb._instruction_addresses.extend([vn.offset for vn in op.inputs])
                     last_decode_addr = op.inputs[0].offset

